project/creditor/urls.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from django.conf.urls import url
+from django.contrib.auth.decorators import permission_required
+
+from . import views
+
+urlpatterns = [
+    url(r'^transaction_table/$', permission_required('is_staff')(views.TransactionYearView.as_view()), name="table_y"),
```

Comment:
This decorator redirects to nonexistent URL for login, probably a missing configuration.